packages/backend/src/open_payments/payment/outgoing/service.test.ts

```diff
@@ -59,6 +59,7 @@ describe('OutgoingPaymentService', (): void => {
   let incomingPayment: IncomingPayment
```

Comment:
I think what we are missing in general in this file (even before this PR) is asserting we have properly published the `OutgoingPaymentEvent` with the correct `data` property. Would be good to test all of the properties explicity.

Reply:
Just to clarify @mkurapov, this would be testing that the values for the returned `incomingPayment` is "ok"? Example:
```
   assert.ok(incomingPayment.id)
      assert.ok(incomingPayment.createdAt)
      assert.ok(incomingPayment.updatedAt)
      assert.ok(incomingPayment.expiresAt)
      assert.ok(incomingPayment.processAt)
      assert.ok(incomingPayment.assetId)
      assert.ok(incomingPayment.asset)
      assert.ok(incomingPayment.asset.code)
      assert.ok(incomingPayment.asset.scale)
      assert.ok(incomingPayment.asset.createdAt)
      assert.ok(incomingPayment.walletAddressId)
      assert.ok(incomingPayment.walletAddress)
      expect(incomingPayment.state).toEqual('PENDING')
      expect(incomingPayment.incomingAmount?.value).toBeGreaterThan(0n)
      assert.ok(incomingPayment.incomingAmount?.assetCode)
      assert.ok(incomingPayment.incomingAmount?.assetScale)
      expect(incomingPayment.receivedAmount.value).toEqual(0n)
      assert.ok(incomingPayment.receivedAmount?.assetCode)
      assert.ok(incomingPayment.receivedAmount?.assetScale)
```

Reply:
@mkurapov Ah, got it! I am updating the test cases to cater for that. 🙇

Reply:
@mkurapov I have added a verification on the data (`OutgoingPaymentEvent`) of the webhook for the client. Resolving this comment.